dev/test/wasm-nextest

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eou pipefail
+
+export RUSTFLAGS="-Ctarget-feature=+bulk-memory,+mutable-globals,+atomics --cfg getrandom_backend=\"wasm_js\"${RUSTFLAGS:=}"
+
+WASM_BINDGEN_TEST_ONLY_WEB=1 \
+  WASM_BINDGEN_TEST_TIMEOUT=180 \
+  CHROMEDRIVER="chromedriver" \
+  cargo nextest run \
```

Comment:
There's currently a bug in wasm-bindgen-test-runner that prevents this from working, but will be nice to have consistent test output between all platforms once the bug is fixed